pages/button/permutations.page.tsx

```diff
@@ -66,6 +69,7 @@ const permutations = createPermutations<ButtonProps>([
     children: ['With custom svg icon'],
     ariaLabel: ['With custom svg icon'],
     variant: ['normal', 'icon', 'inline-icon', 'primary'],
+    badge: [true, false],
```

Comment:
Adding badges to existing set multiplied all existing permutations twice and exceeded the page height limit for screenshot tests. There is a warning on this page:

<img width="954" alt="image" src="https://github.com/cloudscape-design/components/assets/812240/df1558f6-6549-4fa2-9a55-aa54d3b095f2">


We need to create a separate set specifically for badge use-cases below